lib/html_parser.dart

```diff
@@ -387,8 +393,10 @@ class HtmlRichTextParser extends StatelessWidget {
         ));
 
         // if the parent is a normal span, just add this to that list
-      } else {
+      } else if (!(parseContext.parentElement.children is List<Widget>)) {
         parseContext.parentElement.children.add(span);
+      } else {
+        print('doing nothing');
```

Comment:
Can we just make this a comment since it doesn't really affect the average user?

```suggestion
        //Doing nothing
```

Reply:
done.